src/Sarif/Visitors/RebaseUriVisitor.cs

```diff
@@ -109,15 +144,15 @@ public override Run VisitRun(Run node)
         /// If we are changing the URIs in Results to be relative, we need to also change the URI keys in the files dictionary
         /// to be relative.
         /// </summary>
-        /// <param name="node">Result location being changed to relative.</param>
-        internal void RebaseFilesDictionary(PhysicalLocation node)
+        /// <param name="node">File location being changed to relative.</param>
+        internal void RebaseFilesDictionary(FileData node)
```

Comment:
>RebaseFilesDictionary [](start = 22, length = 21)

It took me a while to figure out what this did because of the misleading name. It actually rebases only a single dictionary entry, not the whole dictionary. Please rename to `RebaseFilesDictionaryEntry`.

Reply:
+1

---
In reply to: [218962150](https://github.com/Microsoft/sarif-sdk/pull/1098#discussion_r218962150) [](ancestors = 218962150)